Badge/index.html

```diff
@@ -6,69 +6,137 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link href="./index.css" rel="stylesheet">
+    <link rel="icon" href="../assets/icons/icons8-reactos.svg">
     <link href="./documentation.css" rel="stylesheet">
-    <title>Badge</title>
+    <title>NeonUI | Badge</title>
 </head>
 
 <body>
-    <div class="container">
-        <div class="avatar-badge-container">
-            <div class="avatar avatar-lg">
-                <img loading="lazy" class="img-responsive img-round" src="../assets/images/logo_2.jpg" alt="logo">
-                <span class="status-badge badge-lg badge-online top-right"></span>
+    <div class="menu">
+        <header>
+            <div id="hamburger" class="hamburger-menu">
+                <span></span>
+                <span></span>
+                <span></span>
```

Comment:
These elements are of no use, you can remove them.

Reply:
i have used these span for hamburger icon so these tags are in use